mongodbatlas/fw_resource_mongodbatlas_alert_configuration.go

```diff
@@ -765,6 +848,39 @@ func newTFMetricThresholdConfigModel(matlasMetricThreshold *matlas.MetricThresho
 	return []tfMetricThresholdConfigModel{newState}
 }
 
+func newTFMetricThresholdConfigModelV2(t *admin.ServerlessMetricThreshold, currStateSlice []tfMetricThresholdConfigModel) []tfMetricThresholdConfigModel {
```

Comment:
```suggestion
func newTFMetricThresholdConfigModel(t *admin.ServerlessMetricThreshold, currStateSlice []tfMetricThresholdConfigModel) []tfMetricThresholdConfigModel {
```
Why using v2 in the name?

Reply:
I upgraded data sources for alert_notification and alert_notifications but not for the resource.  These are 3 functions in the resource file which are also being used by the data sources, so I kep the original functions for the resource and these v2 methods can be used when we migrate the alert configuration resource to the new sdk